package.json

```diff
@@ -24,6 +24,7 @@
     "check:modified-files": "yarn workspace @fluentui/scripts just check-for-modified-files",
     "check:affected-package": "node ./scripts/monorepo/checkIfPackagesAffected.js",
     "check:installed-dependencies-versions": "satisfied --skip-invalid --ignore \"prettier|angular|lit|sass|@storybook/web-components|@storybook/html|svelte|@testing-library|vue|@cypress/react\"",
+    "chromatic": "cd apps/public-docsite-v9 && yarn chromatic",
```

Comment:
I'm suggest to not add additional spam to root package.json. this alias is also redundant as this should be done by the workspace task runner that we use as source of truth atm

`yarn workspace @fluentui/public-docsite-v9 chromatic`

Reply:
i was getting ahead of myself as I was thinking I'd already taken the website out of workspaces :)

Reply:
agree this isn't necessary, and with the possibility of multiple sites, it's better to push this into the site folder itself.